tests/bsim/bluetooth/host/l2cap/multilink_peripheral/src/central.c

```diff
@@ -0,0 +1,164 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/kernel.h>
+#include <zephyr/bluetooth/bluetooth.h>
+#include <zephyr/bluetooth/conn.h>
+#include <zephyr/bluetooth/att.h>
+#include <zephyr/bluetooth/l2cap.h>
+#include <zephyr/logging/log.h>
+
+#include "testlib/scan.h"
+#include "testlib/conn.h"
+
+#include "babblekit/flags.h"
+#include "babblekit/sync.h"
+#include "babblekit/testcase.h"
+
+/* local includes */
+#include "data.h"
+
+LOG_MODULE_REGISTER(central, CONFIG_APP_LOG_LEVEL);
+
+static int total_rx;
+static struct bt_l2cap_le_chan le_chan;
+
+void sent_cb(struct bt_l2cap_chan *chan)
+{
+	TEST_FAIL("Tester should not send data");
+}
+
+int recv_cb(struct bt_l2cap_chan *chan, struct net_buf *buf)
+{
+	LOG_DBG("received %d bytes", buf->len);
+	total_rx++;
+
+	return 0;
+}
+
+void l2cap_chan_connected_cb(struct bt_l2cap_chan *chan)
+{
+	LOG_DBG("%p", chan);
+}
+
+void l2cap_chan_disconnected_cb(struct bt_l2cap_chan *chan)
+{
+	LOG_DBG("%p", chan);
+}
```

Comment:
It doesn't matter much, but callbacks can/should be `static`

Reply:
yup missed those. The dut's ones are static.